server/server.go

```diff
@@ -525,15 +529,75 @@ func (s *Server) loadIndex(id string) error {
 	return nil
 }
 
+const repositoryIDSize = sha256.Size
+const RepositoryVersion = 1
+
+func (s *Server) createConfig() (err error) {
+	s.Config.ChunkerPolynomial, err = chunker.RandomPolynomial()
+	if err != nil {
+		return err
+	}
+
+	newID := make([]byte, repositoryIDSize)
+	_, err = io.ReadFull(rand.Reader, newID)
+	if err != nil {
+		return err
+	}
+
+	s.Config.ID = hex.EncodeToString(newID)
+	s.Config.Version = RepositoryVersion
```

Comment:
This is never read... If I go and change the version to 0 or 2 or 17, nothing complains.